components/sidebar_right_menu/sidebar_right_menu.jsx

```diff
@@ -411,17 +411,17 @@ export default class SidebarRightMenu extends React.Component {
         if (this.props.helpLink) {
             helpLink = (
                 <li>
-                    <Link
+                    <a
                         target='_blank'
                         rel='noopener noreferrer'
-                        to={this.props.helpLink}
+                        href={this.props.helpLink}
```

Comment:
Since props.helpLink (and others below) are configurable, we might need to wrap it with `useSafeUrl`.  Maybe something like: `href={useSafeUrl(this.props.helpLink, "https://about.mattermost.com/default-help/")}`.  Any thoughts?

Reply:
Excellent insight! I've updated the PR to leverage `useSafeUrl`, but I'm inclined to leave off the default, as we don't use it in lots of similar cases probably to avoid "unbranding" a customer's customization by accident. If they do happen to configure one of these unsafe urls, it will be obviously broken when the URL is stripped out entirely.

Reply:
Right, I totally agree :)